app/javascript/stylesheets/sponsor-us.css

```diff
@@ -70,20 +70,28 @@
     @apply border-wnbrb-orange-default;
 }
 
+.sponsorship-option-radio {
+    position: relative;
+}
+
 .sponsorship-option-radio input[type='radio'] {
-    display: none;
+    opacity: 0;
 }
 
-.sponsorship-option-radio label {
+.sponsorship-option-label {
     @apply hover:cursor-pointer;
 }
 
-.sponsorship-option-radio input[type='radio'] + label:before {
-    @apply w-4
-      h-4
-      top-[3px]
-      mr-3
-      relative
+.sponsorship-option-title {
+    @apply ml-3;
+}
+
+.sponsorship-option-radio input[type='radio'] + .sponsorship-option-title:before {
```

Comment:
Everything from here on in just re-implements the radio button styles now that I've re-arranged the components a bit.